src/components/Calendar/Section.js

```diff
@@ -36,28 +38,69 @@ export const styles = {
     justifyContent: 'space-between',
     flexWrap: 'wrap',
   },
+  modal: {
+    position: 'absolute',
+    width: '80%',
```

Comment:
This looks extremely wide on a large monitor - I don't think it needs to be this big. Let's just set a max-width of 450px and see how that looks.